gramps/gui/clipboard.py

```diff
@@ -403,8 +404,9 @@ def refresh(self):
             # String must be unicode for truncation to work for non
             # ascii characters
             note = str(note)
```

Comment:
On an unrelated note, what does this comment mean? Is this a Python2-ism? If it were `bytes`, it certainly wouldn't do the right thing on Python 3:
```pycon
>>> b = b'abc'
>>> str(b)
"b'abc'"
```
It seems like both the command and `str` cast should be removed (either that or it needs to _conditionally_ `decode`.)

Reply:
I don't know. it is a very old comment I think.

Reply:
After looking at the source, b is a StyledText, so str is needed to get the text.